modules/ROOT/pages/getting-started.adoc

```diff
@@ -1,142 +1,67 @@
 :experimental:
 = Fedora CoreOS - Getting Started
 
-Fedora CoreOS has no install-time configuration. Every Fedora CoreOS system begins with a generic, unconfigured disk image. On first boot Ignition will read the supplied config and configure the system. Ignition configs are usually supplied via the cloud’s userdata mechanism, or, in the case of bare metal, injected at install time. This guide will show you how to launch Fedora CoreOS on AWS, QEMU, and bare metal as well as how to create Ignition configs.
+== Background
+
+Fedora CoreOS (FCOS) has no install-time configuration. Every FCOS system begins with a generic disk image. For each deployment mechanism (bare metal, local VM, or cloud VM), the configuration must be supplied at first boot. This is done via a configuration file and FCOS Ignition. For cloud deployments, Ignition supplies the configuration via the cloud’s userdata mechanism. In the case of bare metal, Ignition injects the configuration at install time.
+
+This guide will show you how to launch Fedora CoreOS on AWS, QEMU, and bare metal.
+
+=== Prerequisite
+Before you can launch FCOS, you must have an Ignition configuration file. To produce an Ignition file, see xref:producing-ign.adoc[Producing an Ignition File].
 
 == Launching Fedora CoreOS
 
 === Booting on AWS
 
-To boot on AWS, find the correct AMI in the https://getfedora.org/coreos/download/[download page] and specify the Ignition config as the https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-instance-metadata.html#instancedata-add-user-data[user-data]. Note that while the AWS documentation mentions cloud-init and scripts, Fedora CoreOS only accepts Ignition configs; it does not support cloud-init or running scripts from userdata.
+Download the correct AMI in the https://getfedora.org/coreos/download/[download page].
+
+Then, specify the Ignition configuration file as the https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-instance-metadata.html#instancedata-add-user-data[user-data].
+
+NOTE: While the AWS documentation mentions cloud-init and scripts, FCOS accepts only Ignition configuration files; it does not support cloud-init or running scripts from user-data.
 
 .Example launching Fedora CoreOS on AWS
 [source, bash]
 ----
-aws ec2 run-instances <other options> --user-data file://config.ign
+aws ec2 run-instances <other options> --user-data file://example.ign
 ----
 
 === Booting with QEMU
+Go to the https://getfedora.org/coreos/download/[download page] to retrieve the latest FCOS qcow file.
```

Comment:
s/qcow file/image suitable for QEMU/.

(rationale: we'll likely end up with several qcow files, but only one relevant for QEMU).